Kernel/Storage/StorageManagement.cpp

```diff
@@ -445,14 +439,11 @@ UNMAP_AFTER_INIT void StorageManagement::initialize(StringView root_device, bool
     } else {
         enumerate_pci_controllers(force_pio, poll);
     }
-    // Note: Whether PCI bus is present on the system or not, always try to attach
-    // a given ramdisk.
-    auto controller = RamdiskController::try_initialize();
-    if (controller.is_error()) {
-        dmesgln("Unable to initialize RAM controller: {}", controller.error());
-    } else {
-        m_controllers.append(controller.release_value());
+
+    if (auto sdhc = SDHostController::try_initialize(); !sdhc.is_error()) {
```

Comment:
maybe debug-print an error?

Reply:
Good idea. Decided to print an error via `dmesgl` if `try_initialize` returns an error and it's not because there's no sd card slot